client/components/dialog/index.jsx

```diff
@@ -59,15 +59,15 @@ var Dialog = React.createClass( {
 			transitionName = this.props.baseClassName || 'dialog';
 
 		ReactDom.render(
-			<SingleChildCSSTransitionGroup
+			<ReactCSSTransitionGroup
 				transitionName={ transitionName }
 				component="div"
 				transitionLeave={ this.props.transitionLeave }
-				onChildDidLeave={ this.onDialogDidLeave }
-				enterTimeout={ this.props.enterTimeout }
-				leaveTimeout={ this.props.leaveTimeout }>
+				componentDidLeave={ this.onDialogDidLeave }
```

Comment:
True, the call tree in https://github.com/facebook/react/blob/master/src/addons/transitions/ReactTransitionGroup.js is rather complex. If I've groked it correctly from a quick glance, we might have to attach that prop to the child `dialogComponent` instead of the wrapping `ReactCSSTransitionGroup`...